lib/features/home/widgets/googe_map.dart

```diff
@@ -29,15 +29,33 @@ class _GoogleMapViewState extends State<GoogleMapView>
   late final DrawableRoot markerAddDrawableRoot;
 
   late final AnimationController _controller;
+  late GoogleMapController mapController;
+  late ScreenCoordinate coordinate;
+  bool isShow = false;
+
+  Future<void> startAnimationMarker(LatLng latLng) async {
+    mapController
+        .animateCamera(CameraUpdate.newCameraPosition(
+            CameraPosition(target: latLng, zoom: 15)))
+        .then((_) async {
+      _computeMarker(
+        angleInDegree: _controller.value * 45,
+      );
+    });
+  }
 
   @override
   void initState() {
     super.initState();
 
     _controller = AnimationController(
-        vsync: this, duration: const Duration(milliseconds: 300))
+        vsync: this, duration: const Duration(milliseconds: 500))
       ..addListener(() {
-        _computeMarker(angleInDegree: _controller.value * 45);
+        LatLng latLng = LatLng(
```

Comment:
`widget.currentLocation` is a LatLng object, you don't need to do this conversion